server/src/main/java/io/unitycatalog/server/utils/ServerProperties.java

```diff
@@ -149,44 +180,46 @@ public Map<String, ADLSStorageConfig> getAdlsConfigurations() {
    * <ol>
    *   <li>System property
    *   <li>Environment variable
+   *   <li>server.properties file
    * </ol>
+   *
+   * <p>If it's not set, the default value is returned.
    */
-  public String getProperty(String key) {
-    if (System.getProperty(key) != null) return System.getProperty(key);
-    if (System.getenv().containsKey(key)) return System.getenv(key);
-    return properties.getProperty(key);
+  public String get(Property property) {
+    return getProperty(property.key);
   }
 
-  public String get(Property property) {
-    return getProperty(property.getKey());
+  /** Get a property value by key name. */
+  private String getProperty(String key) {
+    if (System.getProperty(key) != null) {
+      return System.getProperty(key);
+    }
+    if (System.getenv().containsKey(key)) {
+      return System.getenv(key);
+    }
+    // Finally try properties. If not found in properties, this would return default value or null
+    // if no default value.
+    return properties.getProperty(key);
   }
 
   public void set(Property property, String value) {
-    properties.setProperty(property.getKey(), value);
+    properties.setProperty(property.key, value);
   }
 
-  /**
-   * Get a property value by key with a default value
-   *
-   * @see Properties#getProperty(String key, String defaultValue)
-   */
-  public String getProperty(String key, String defaultValue) {
-    String val = getProperty(key);
-    return (val == null) ? defaultValue : val;
+  private boolean isTrueOrEnable(String value) {
```

Comment:
Seems like a method with combinated meanings , which some kind of confusing me.  For every variable, either we use the `true` or `enable` ?  will that be too relax for end users ?  Why not just define then as boolean with `true` or `false` option.  Let's just remove the `enable` or `disable` option ?

Reply:
Yes,  I think it's good to do this in the next PR. Thanks @yili-db for the clarfication !